bindings/libdnf/conf.i

```diff
@@ -6,10 +6,21 @@
 %module "libdnf/conf"
 #endif
 
+%include <exception.i>
 %include <std_vector.i>
 
 %import "common.i"
 
+%exception {
```

Comment:
I am not 100% sure whether it is a good idea but what about to move it to shared part of code like common.i?

Reply:
@j-mracek I'm not sure if that's possible. I tried it without success. Exception handling defined in the "common.i" file only worked in the "common.i" file. Not in those that import it.